src/package-styles.css

```diff
@@ -0,0 +1,9 @@
+input, select {
+  @apply dark:tce-text-slate-600
+}
+
+@media (prefers-color-scheme: dark) {
+  .apexcharts-legend-text {
+    @apply !tce-text-slate-50
```

Comment:
Why do we need '! important' here?

Reply:
I think because the class is used so far down the hierarchy and the chart has some method of setting the colour already, we need to override it. Doesn't work without it and any other methods get into conditionally setting things on the chart options in javascript etc.

Reply:
Ok, maybe !important is the most maintainable solution then.